submissions/BogdanGapon/HTML & CSS popup/style.css

```diff
@@ -0,0 +1,235 @@
+  position: absolute;
+  -webkit-appearance: none;
+     -moz-appearance: none;
+          appearance: none;
+}
+
+.custom-checkbox {
+  position: relative;
+}
+.custom-checkbox::before {
+  content: " ";
+  display: -webkit-inline-box;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  width: 30px;
+  height: 30px;
+  background-image: url(../img/popup-button.png);
+  background-repeat: no-repeat;
+  background-size: 30px 30px;
+  cursor: pointer;
+}
+
+.popup-wrapper {
+  display: none;
+  position: absolute;
+  top: 65px;
+  right: 50px;
+  width: 350px;
+  height: 400px;
+  padding: 30px 15px 30px;
+  border: 1px solid rgba(128, 128, 128, 0.4);
+  -webkit-box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
+          box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
+  background-color: #fff;
+  overflow: auto;
+}
+.checkbox:checked ~ .popup-wrapper {
+  display: block;
+}
+
+.popup-flex-wrapper {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+      flex-wrap: wrap;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  margin-bottom: 10px;
+  gap: 40px;
+}
+
+.popup-wrapper img {
+  margin-bottom: 10px;
+}
+
+.span {
+  content: " ";
+  position: absolute;
+  top: 55px;
+  z-index: 10;
+  display: none;
+  width: 20px;
+  height: 20px;
+  border-top: 1px solid rgba(128, 128, 128, 0.4);
+  border-left: 1px solid rgba(128, 128, 128, 0.4);
+  -webkit-transform: rotate(45deg);
+          transform: rotate(45deg);
+  background-color: #fff;
+}
+.checkbox:checked + .span {
+  display: block;
+}
+
+.popup-list__link {
+  width: calc((100% - 120px) / 3);
+  text-decoration: none;
+  color: #000;
+  text-align: center;
+}
+.popup-list__link:hover, .popup-list__link:focus {
+  border: 1px solid grey;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  -o-object-fit: contain;
+     object-fit: contain;
+}
+
+.label-more {
+  position: absolute;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  width: 100%;
+  height: 40px;
+  left: 0;
+  z-index: 1;
+  background-color: bisque;
+  cursor: pointer;
+}
+
+.next-popup-wrapper {
+  display: none;
+  position: absolute;
+  top: 500px;
+  right: 0px;
+  padding: 30px 15px 50px;
+  background-color: #fff;
+  z-index: 10;
+}
+.checkbox-more:checked ~ .next-popup-wrapper {
+  display: block;
+}
+
+.checkbox-more:checked + .is-hidden {
+  visibility: hidden;
+  pointer-events: none;
+}/*# sourceMappingURL=style.css.map */
```

Comment:
Please, avoid unnecessary comments and add white-space in the end of the file - [reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)